therapy/schemas.py

```diff
@@ -358,7 +383,118 @@ def schema_extra(schema: Dict[str, Any],
                 schema.pop('title', None)
             for prop in schema.get('properties', {}).values():
                 prop.pop('title', None)
-            schema['example'] = {}  # TODO
+            schema['example'] = {
+                "query": "cisplatin",
+                "warnings": None,
+                "match_type": 80,
+                "value_object_descriptor": {
+                    "id": "normalize.therapy:cisplatin",
+                    "type": "TherapyDescriptor",
+                    "value": {
+                        "type": "Therapy",
+                        "therapy_id": "rxcui:2555"
```

Comment:
We should make the `id` change in our value object for all normalizers. We can add this in a new issue though. 
```suggestion
                        "id": "rxcui:2555"
```